cmd/thanos/store.go

```diff
@@ -14,6 +14,9 @@ import (
 	"github.com/pkg/errors"
 	"github.com/prometheus/client_golang/prometheus"
 	"github.com/prometheus/prometheus/pkg/relabel"
+	"gopkg.in/alecthomas/kingpin.v2"
+	yaml "gopkg.in/yaml.v2"
+
```

Comment:
let's remove blank line

Reply:
Done. I would suggest you to add a linter on imports so that it's easier to enforce your preferred styling 😉

Reply:
We do have a ticket for that https://github.com/thanos-io/thanos/issues/984, I think none of the linters supports this atm

Reply:
Yes, extra new lines are tricky. Golangci-lint can check for imports ordering and grouping at least.